### PR TITLE
Wrapping paper fixes

### LINKED ISSUE #6840: Cannot put wrapping paper inside a backpack or any other wrappable container
### Environment
- Game Version: latest

### Detailed description of bug

if you try to place a roll of wrapping paper inside a backpack on help intent, you begin to wrap the backpack, even if the backpack is equipped. If you use any other intent, nothing happens.

### Steps to reproduce

1. get a roll of wrapping paper
2. try to put it in the backpack
3. observe how none of the intents have the desired effect of stowing the wrapping paper in the backpack.

### PR DESCRIPTION
**Pull request by AtnerNT**
A few fixes relating to wrapping paper:
CL: [Fix] Fixes Wrapped Crates and Lockers not being able to be pushed nor pulled.
CL: [Fix] Players are no longer able to wrap items that are equipped like backpacks, instead they must be taken off to be wrapped, fixing the annoying issue of wrapping your backpack when trying to stow items.
Replaces the wrapped locker sprite with one that correctly matches the dimensions of unitystation's lockers instead of tg's.
(Thanks to BlitzGunner225 for the sprite)

### CODE REVIEW

**Review by ThatDan123 — DISMISSED** [review]

**Comment by AtnerNT** [comment]
Just a clarification, this makes it so you can only wrap items that are on the ground or in your hands. If you have your backpack equipped it wont be wrapped as it is in your back slot, instead you will simply stow the wrapping paper.

**Comment by PerfectTangent** [comment]
> If you have your backpack equipped it wont be wrapped as it is in your back slot, instead you will simply stow the wrapping paper.

big brain fix.

File: UnityProject/Assets/Scripts/Items/Cargo/Wrapping/WrappingPaper.cs [hunk 1]
```diff
@@ -45,6 +45,9 @@ public void ServerPerformInteraction(HandApply interaction)
 
 		public bool WillInteract(InventoryApply interaction, NetworkSide side)
 		{
+			//interaction only occurs if wrapping target is on a hand slot, prevents player from wrapping clothes they are wearing.
+			if (!interaction.IsToHandSlot) return false;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Items/Cargo/Wrapping/WrappingPaper.cs]:
```cs
return interaction.IsToHandSlot &&
           DefaultWillInteract.Default(interaction, side) &&
           CommonWillInteract(interaction, side)
```

**Comment by PerfectTangent** [comment]
the only issue I see is that it doesn't solve the issue of trying to stow wrapping paper into containers that aren't being worn, such as boxes, so it doesn't completely solve the problem. Still, this is an improvement.

**Comment by AtnerNT** [comment]
Don't merge rn, just waiting on the finalized sprite

**Comment by NoooneyDude** [comment]
What's wrong with the current sprite change?

**Comment by corp-0** [comment]
yeah it was good to be merged before my logic nitpick. What is it you want to do before merging?

**Comment by AtnerNT** [comment]
Ah just very minor changes to the sprite to match the other parcel sprites- mainly colour changes- should be good to merge now

**Review by corp-0 — APPROVED** [review]

**Review by ThatDan123 — APPROVED** [review]

**Pull request merged**